src/liberror/specification

```diff
@@ -660,3 +660,93 @@ description:python warning
 severity:warning
 ingroup:plugin
 module:python
+
+number:113
+description:couldnt open file
+severity:error
+ingroup:plugin
+module:lineendings
+
+number:114
+description:invalid line ending
+severity:error
+ingroup:plugin
+module:lineendings
+
+number:115
+description:inconsistent line endings
+severity:error
+ingroup:plugin
+module:lineendings 
+
+number:116
+description: couldn't open file
+severity:error
+ingroup:plugin
+module:csvstorage
+
+number:117
+description:invalid number of columns
+severity:error
+ingroup:plugin
+module:csvstorage
+
+number:118
+description:problem during parsing of csvfile (see reason)
+severity:warning
+ingroup:plugin
+module:csvstorage
+
+number:119
+description:out of memory
+severity:error
+ingroup:plugin
+module:csvstorage
+
+number:120
+description: Couldn't compile regex
+severity: error
+ingroup:plugin
+module:enum
+
+number:121
+description:Validation failed
+severity:error
+ingroup:plugin
+module:enum
+
+number:122
+description:not a valid Polish prefix notation syntax
+severity:error
+ingroup:plugin
+module:calculate
+
+number:123
+description:invalid value
+severity:error
+ingroup:plugin
+module:calculate
+
+number:124
+description:key doesn't exist
+severity:error
+ingroup:plugin
+module:calculate
+
+number:125
+description:Key doesn't exist
+severity:error
+ingroup:plugin
+module:conditionals
+
+number:126
+description:Syntax error
+severity:error
+ingroup:plugin
+module:conditionals
+
+number:127
+description:Validation failed
```

Comment:
It would be good to have a possibility to check if any type of validation failed. We could use some inheritance schema for that.

See #307